lib/tokeniser.js

```diff
@@ -251,3 +251,6 @@ class WebIDLParseError extends Error {
     this.tokens = tokens;
   }
 }
+module.exports={
```

Comment:
This is old CommonJS way to export things, here instead we want to use more modern way: just add `export` before `class WebIDLParseError` 👍